app/src/main/java/com/example/admin/personallibrarycatalogue/AddBookActivityFragment.java

```diff
@@ -109,10 +111,25 @@ public void onClick(View v) {
                 EditText descriptionEditText = (EditText) rootView.findViewById(R.id.description_edit_text);
                 String description = descriptionEditText.getText().toString();
 
+                EditText yearEditText = (EditText) rootView.findViewById(R.id.year_edit_text);
+
+                int year = 0;
+
+                if (yearEditText.getText().toString().length() > 0) {
+                    try {
+                        year = Integer.parseInt(yearEditText.getText().toString());
+                    } catch (NumberFormatException e) {
+                        Toast.makeText(getActivity(), "Input is not an integer!", Toast.LENGTH_SHORT).show();
+                    }
+                }
```

Comment:
Mental pabulum:
Do we need check for non empty string if we already have try-catch with NumberFormatException?